netboot-oci-artifacts.md

```diff
@@ -113,6 +113,51 @@ An image manifest containing Netboot files MUST be tagged in the format of ``ver
   [GOARCH](https://go.dev/doc/install/source#environment).
 
 
+## Image index containing OCI artifacts
+
+For the multi-arch builds it is possible to create [OCI image index](https://github.com/opencontainers/image-spec/blob/main/image-index.md)
+that points to specific image manifests, ideal for one or more platforms.
+
+Every entry of image manifest MAY specify additional OPTIONAL metadata via annotations.
```

Comment:
we can specify here examples

Reply:
Well, I would probably actually name them: name and version. Can add later.